blockchain/chain_sync/src/tipset_syncer.rs

```diff
@@ -215,7 +225,22 @@ impl TipsetGroup {
     }
 
     fn is_heavier_than(&self, other: &Self) -> bool {
-        self.heaviest_weight() > other.heaviest_weight()
+        self.cmp(other).is_gt()
+    }
+
+    fn cmp(&self, other: &Self) -> Ordering {
```

Comment:
Should this be rather an implementation for `PartialOrd`? It would simplify some logic in other places, e.g. we wouldn't have to call `max_by` with a lambda, just `max`.

Reply:
Yes, I see your point. In this case I would have also to implement `PartialEq` [trait](https://doc.rust-lang.org/std/cmp/trait.PartialEq.html) (not sure how). So don't makes sense to go down to this route, feels a bit too much complex for a seldom use case.